index.html

```diff
@@ -206,40 +256,68 @@ <h4>
             <dd>
               USB Product ID
             </dd>
+            <dt>
+              <dfn>bluetoothServiceClassId</dfn> member
+            </dt>
+            <dd>
+              Bluetooth service class ID
+            </dd>
           </dl>
           <p>
-            A serial port <dfn data-lt="match the filter">matches the
-            filter</dfn> |filter:SerialPortFilter| if these steps return
-            `true`:
+            A serial port |port:SerialPort| <dfn>matches the filter</dfn>
+            |filter:SerialPortFilter| if these steps return `true`:
           </p>
           <ol>
+            <li>Let |info:SerialPortInfo| be the result of calling
+            |port|.{{SerialPort/getInfo()}}.
+            </li>
+            <li>If |filter|["{{SerialPortFilter/bluetoothServiceClassId}}"] is
+            present:
+              <ol>
+                <li>If the serial port is not part of a Bluetooth device,
+                return `false`.
+                </li>
+                <li>If |filter|["{{SerialPortFilter/bluetoothServiceClassId}}"]
+                is equal to
+                |info|["{{SerialPortInfo/bluetoothServiceClassId}}"], return
+                `true`.
+                </li>
+                <li>Otherwise, return `false`.
+                </li>
+              </ol>
+            </li>
             <li>If |filter|["{{SerialPortFilter/usbVendorId}}"] is not present,
             return `true`.
             </li>
             <li>If the serial port is not part of a USB device, return `false`.
             </li>
-            <li>If the USB device's vendor ID is not equal to
+            <li>If |info|["{{SerialPortInfo/usbVendorId}}"] is not present,
```

Comment:
The code above assumes that if `port` is a Bluetooth device then `info["bluetoothServiceClassId"]` exists. I think we can do the same for USB devices.

Reply:
Removed the `info["usbVendorId"]` check